src/scheduler/affinity.rs

```diff
@@ -0,0 +1,59 @@
+use super::worker::ThreadId;
+use crate::util::options::AffinityKind;
+use libc::{cpu_set_t, sched_getaffinity, sched_setaffinity, CPU_COUNT, CPU_SET, CPU_ZERO};
```

Comment:
The import should also be guarded for Linux only.

Reply:
True. Thanks for the catch.